src/services/GraphApplication.ts

```diff
@@ -98,6 +98,17 @@ export class GraphApplication {
        Example: if the user asks for an ordered list of revenue based on user id, try to generate a query like this: select "USER_ID", "NAME", sum(cast("REVENUE" as numeric)) as total_revenue from "snowflake_OFFER_CHECKOUT" group by "USER_ID", "NAME", "REVENUE" order by total_revenue desc limit 10;`,
        toolFunction: clientAgentFunction,
       },
+      createTableStructure: {
+        agent: createAgent(strongestModel, [createTableStructure], true),
+        agentPrompt: `You are an LLM specialized in the entire process of transforming JSON data into a fully functional PostgreSQL. This is done by using your createTableStructure tool to create the table. This should return the column name followed by the data type of that column.
+          The response should always have this structure and include the columns names and types like in this example: 
```

Comment:
We can't give examples that contain formats, the format is defined at the tool level, we need to explain them with words, otherwise some models get confused, we'll review this in our session tomorrow

Reply:
Ok, removed